easybuild/easyconfigs/u/UCX/UCX-1.16.0-rc4-GCCcore-11.2.0.eb

```
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.16.0-rc4'
+
+homepage = 'https://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openucx/ucx/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.gz']
```

@hajgato The tarball is named `ucx-1.16.0.tar.gz`, so we'll need something like this:

```suggestion
sources = [{'download_filename': 'ucx-%s.tar.gz' % version.split('-'), 'filename': SOURCELOWER_TAR_GZ}]